### PR TITLE
Send email to trainees when assign or remove of course

### PR DESCRIPTION
**Pull request by ngocquyhoang**

### CODE REVIEW

File: app/models/user_course.rb [hunk 1]
```diff
@@ -25,4 +29,12 @@ def all_activities
   def delete_activity
     Activity.activity_courses(self).destroy_all
   end
+
+  def send_assign_email
+    Mailer.assign_trainee_email(self.user, self.course).deliver_now
```
Comment (tanphan313) [inline app/models/user_course.rb]: remove self

File: app/models/user_course.rb [hunk 2]
```diff
@@ -25,4 +29,12 @@ def all_activities
...
+  end
+
+  def send_remove_email
+    Mailer.remove_trainee_email(self.user, self.course).deliver_now
```
Comment (tanphan313) [inline app/models/user_course.rb]: như trên

File: config/environments/development.rb [hunk 3]
```diff
@@ -38,4 +38,7 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.active_record.dump_schema_after_migration = false
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.default_url_options = { host: "localhost:3000" }
```
Comment (tanphan313) [inline config/environments/development.rb]: space

**Comment by tanphan313** [comment]
ok

File: config/environment.rb [hunk 4]
```diff
@@ -1,5 +1,15 @@
 # Load the Rails application.
 require File.expand_path('../application', __FILE__)
 
+ActionMailer::Base.smtp_settings = {
```
Comment (dieunb) [inline config/environment.rb]: convention

**Pull request merged**